Form/WidgetFilterType.php

```diff
@@ -35,7 +35,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $choices = [];
         foreach ($options['filters'] as $filter) {
             /* @var BaseFilter $filter */
-            $choices['widget_filter.'.$filter->getName()] = $filter->getName();
+            $choices['widget_filter.'.$filter->getName()] = get_class($filter);
```

Comment:
get_class is not reliable because of doctrine proxies.
It should be safer to use something like
```
$em->getClassMetadata(get_class($filter))->getName();
```